pages/[profileSlug]/schemas/[contentSlug]/versions/index.tsx

```diff
@@ -0,0 +1,144 @@
+import React from "react";
+import { useCallback, useEffect, useState } from "react";
+import { GetServerSideProps } from "next";
+import api from "next-rest/client";
+import { Button, majorScale, Pane, Table, Text, toaster } from "evergreen-ui";
+
+import { SchemaPageFrame } from "components";
+import { SchemaPageHeaderProps } from "components/SchemaPageFrame/SchemaPageFrame";
+import { getSchemaPageHeaderData, getSchemaPagePermissions } from "utils/server/schemaPages";
+import { schemaVersonPageSize } from "utils/shared/schemas/versions";
+
+type SchemaPageParams = {
+	profileSlug: string;
+	contentSlug: string;
+};
+
+interface SchemaOverviewProps {
+	schemaPageHeaderProps: SchemaPageHeaderProps;
+}
+
+export const getServerSideProps: GetServerSideProps<SchemaOverviewProps, SchemaPageParams> = async (
+	context
+) => {
+	const { profileSlug, contentSlug } = context.params!;
+	const schemaPageHeaderProps = await getSchemaPageHeaderData(profileSlug, contentSlug);
+	const hasAccess = getSchemaPagePermissions(context, schemaPageHeaderProps);
+	if (!schemaPageHeaderProps || !hasAccess) {
+		return { notFound: true };
+	}
+
```

Comment:
If we're making /versions its own page we may as well fetch the first ten in here so that the page doesn't flicker right away